sims/quick_plot.py

```diff
@@ -78,37 +85,36 @@
                         'engine', 'N', 'rho'], how='outer', suffixes=('_arg', '_noarg')).dropna()
 joined['speedup'] = joined['time_noarg'] / joined['time_arg']
 groups = joined.groupby(['engine', 'N'])
-fig, (ax_fwdpp, ax_simupop) = plt.subplots(
-    2,  sharex=True, sharey=False)
+fig, (ax_fwdpp) = plt.subplots(
+    1,  sharex=True, sharey=False)
 for name, group in groups:
     lstyle = 'solid'
     if name[0] == 'fwdpy11':
         ax = ax_fwdpp
     m = points[name[1]]
     if name[0] == 'simuPOP':
-        ax = ax_simupop
+        continue
     mfacecolor = colors[name[1]]
     popsize = int(name[1])
     ax.plot(group.rho, group.speedup, marker=m,
             ms=4, linestyle=lstyle, label=r'$N = {:.0e}$'.format(popsize),
             color=colors[name[1]],
             markerfacecolor=mfacecolor)
 
-for ax in (ax_fwdpp, ax_simupop):
+for ax in (ax_fwdpp,):
     ax.set_ylabel("Speedup due to\npedigree recording")
 ax_fwdpp.set_title("fwdpy11", fontsize='medium')
```

Comment:
delete this line?

Reply:
w/o it there is no y label

Reply:
I mean line 106
`ax_fwdpp.set_title("fwdpy11", fontsize='medium')` which just removes the title, I think? Anyway that's what I meant